chains/evm/cli/bridge/query-resource.go

```diff
@@ -9,18 +9,18 @@ import (
 
 var queryResourceCmd = &cobra.Command{
 	Use:   "query-resource",
-	Short: "Query the contract address",
-	Long:  "Query the contract address with the provided resource ID for a specific handler contract",
+	Short: "Query the resource ID for a handler contract",
+	Long:  "The query-resource subcommand queries the contract address with the provided resource ID for a specific handler contract",
 	PreRun: func(cmd *cobra.Command, args []string) {
 		logger.LoggerMetadata(cmd.Name(), cmd.Flags())
 	},
 	Run: queryResource,
 }
 
 func BindQueryResourceFlags(cmd *cobra.Command) {
-	cmd.Flags().StringVar(&Handler, "handler", "", "handler contract address")
-	cmd.Flags().StringVar(&ResourceID, "resourceId", "", "resource ID to query")
-	flags.MarkFlagsAsRequired(cmd, "handler", "resourceId")
+	cmd.Flags().StringVar(&Handler, "handler", "", "Handler contract address")
+	cmd.Flags().StringVar(&ResourceID, "resource-id", "", "Resource ID to query")
```

Comment:
I think `-id` in resource is obvious that it is resource.
Also, I think for most cli flags `-id` seems kinda obvious.
Like domain-id, token-id, resource-id. It is obvious if it is only --domain, --resource, --token.

Reply:
So it would be:

`
cmd.Flags().StringVar(&ResourceID, "resource", "", "Resource ID to query")
`
I don't think we lose any information, but it is a bit prettier.
Just my 2 cents